src/app/compressed-air-assessment/compressed-air-banner/compressed-air-banner.component.html

```diff
@@ -100,6 +191,51 @@
                 Scenarios</button>
         </div>
     </div>
-</div>
-
 
+    <div class="d-lg-none d-flex flex-column flex-fill">
+        <div class="d-flex d-lg-none flex-fill align-items-center justify-content-center tabs subtabs compressed-air"
+            [ngClass]="{'br-color-white': selectedModification}">
+            <div class="mod-name active border-right flex-fill" *ngIf="selectedModification">
+                <a>{{selectedModification.name}}
+                    <br>
+                    <span class="help-text">Selected Scenario</span>
+                </a>
+            </div>
+            <div class="mod-name border-right flex-fill" *ngIf="!selectedModification">
+                <a>&mdash; &mdash;
+                    <br>
+                    <span class="help-text">Selected Scenario</span>
+                </a>
+            </div>
+            <div class="pl-2 pr-2 flex-fill">
+                <button class="btn btn-primary" (click)="selectModification()" [disabled]="!selectedModification">View / Add
+                    Scenarios</button>
+            </div>
+        </div>
+        
+        <ul class="pagination d-flex d-lg-none justify-content-between tabs subtabs progress-tabs flex-fill compressed-air">
+            <li class="page-item" [ngClass]="{'disabled': secondaryAssessmentTab == 'modifications'}">
+                <a class="nav-link small" (click)="backAssessmentTab()" [ngClass]="{'disabled': secondaryAssessmentTab == 'modifications'}">
+                    <span class="fa fa-chevron-left"></span></a>
+            </li>
+            <li class="page-item" [ngClass]="{'active': secondaryAssessmentTab == 'modifications'}" *ngIf="secondaryAssessmentTab == 'modifications'">
+                <a class="nav-link small" (click)="changeSecondaryAssessmentTab('modifications')">Setup Profile</a>
+            </li>
+            <li class="page-item" [ngClass]="{'active': secondaryAssessmentTab == 'table', 'disabled': !selectedModification}" *ngIf="secondaryAssessmentTab == 'table'">
+                <a class="nav-link small" (click)="changeSecondaryAssessmentTab('table')">Profile Summary Table</a>
+            </li>
+            <li class="page-item" [ngClass]="{'active': secondaryAssessmentTab == 'graphs', 'disabled': !selectedModification}" *ngIf="secondaryAssessmentTab == 'graphs'">
+                <a class="nav-link small" (click)="changeSecondaryAssessmentTab('graphs')">Profile Summary Graphs</a>
+            </li>
+            <li class="page-item" [ngClass]="{'disabled': secondaryAssessmentTab == 'graphs' || !selectedModification}">
+                <a class="nav-link small" (click)="continueAssessmentTab()" [ngClass]="{'disabled': secondaryAssessmentTab == 'graphs' || !selectedModification}">
+                    <span class="fa fa-chevron-right"></span></a>
+            </li>
+        </ul>
+        
+        <div class="alert alert-info d-lg-none p-3 m-2 small">
```

Comment:
I like that we're messaging them. We can condense this to:
A - "This page is optimized to be viewed on a tablet-sized device or larger" 
OR B - Find the closest standardized device resolution. I think we use 1024) and do "This page is optimized to be viewed on a device with ___ x ____ resolution or larger"

Should also be a shared component or some other angular mechanism to not have to change 5 locations. This would also actually be a great message to have in some of our other strained mobile views. Intern to do vibes.